app/src/main/java/a3/com/convo/fragments/GameFragment.java

```diff
@@ -52,19 +60,33 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
     public void onViewCreated(View view, Bundle savedInstanceState) {
         cardStack = (SwipeDeck) view.findViewById(R.id.cardStack);
+        tvTimer = (TextView) view.findViewById(R.id.tvTimer);
+        timer = new CountDownTimer(120000, 1000) {
+            @Override
+            public void onTick(long l) {
+                // TODO: make this pretty
+                tvTimer.setText(l / 100000 + ":" + l / 1000);
+            }
+
+            @Override
+            public void onFinish() {
+                tvTimer.setText("Game over!");
```

Comment:
Move this string to strings.xml and access from there. This will help us in future for translations.